### ISSUE TITLE
Is your package working?

### ISSUE THREAD

**Issue by pycckuu** [Antriel/atom-thesaurus#1]
I wanted to install the package but haven't found it in Atom database.

**Comment by Antriel** [comment]
Yes, it does work. It's quite simple script really, based on the official spell check package.
The uploading to atom DB via CLI failed without proper error message and I did not have time to track down why.
You can just download it here and add it to atom manually.
